packages/importer/src/utils.ts

```diff
@@ -58,36 +73,64 @@ export function guessType(value: string) {
   } else if (value.indexOf('\n') >= 0) {
     return 'text';
   }
+
   return 'string';
 }
 
-export function castType(a: string, b: string) {
-  if (a === b) {
-    return a;
-  } else if ((a === 'float' && b === 'int') || (a === 'int' && b === 'float')) {
+export function castType(schemaType: string, valueType: string) {
+  if (schemaType === valueType) {
+    return schemaType;
+  } else if ((schemaType === 'float' && valueType === 'int') || (schemaType === 'int' && valueType === 'float')) {
     return 'float';
-  } else if (a === 'text' || b === 'text') {
+  } else if (schemaType === 'text' || valueType === 'text') {
     return 'text';
+  } else if (schemaType === 'link') {
+    return 'link';
+  } else if (schemaType === 'datetime') {
+    return 'datetime';
+  } else if (schemaType === 'bool' && valueType === 'int') {
+    return 'bool';
+  } else if (schemaType === 'multiple' && valueType === 'string') {
+    return 'multiple';
+  } else if (schemaType === 'int' && valueType === 'string') {
+    return 'int';
+  } else if (schemaType === 'float' && valueType === 'string') {
+    return 'float';
   }
+
   return 'string';
 }
 
-export function parseRow(values: string[], types: string[]) {
+export function parseRow(values: string[], types: string[], nullValues: string[] = []) {
   return values.map((val, i) => {
     const type = types[i];
-    if (type === 'int') {
-      const num = +val;
+    const num = +val;
+
+    if (nullValues.includes(val)) {
+      return null;
+    } else if (type === 'int') {
       return Number.isSafeInteger(num) && val !== '' ? num : null;
     } else if (type === 'float') {
-      const num = +val;
       return Number.isFinite(num) && val !== '' ? num : null;
     } else if (type === 'bool') {
-      return ['true', 'false'].includes(val) ? val === 'true' : null;
+      if (val === 'true' || num === 1) {
+        return true;
+      } else if (val === 'false' || num === 0) {
+        return false;
+      } else {
+        return null;
+      }
     } else if (type === 'multiple') {
       return parseArray(val);
     } else if (type === 'email') {
       return val || null;
+    } else if (type === 'link') {
+      return val ? String(val) : null;
+    } else if (type === 'datetime') {
+      const date = new Date(val);
+      return !isNaN(date.getTime()) ? date : null;
```

Comment:
```suggestion
      return isNaN(date.getTime()) ? null : date;
```

Nitpick. You can completely ignore this suggesion 😄